src/Card/CardImageCap.jsx

```diff
@@ -45,26 +46,37 @@ const CardImageCap = React.forwardRef(({
     );
   }
 
-  const handleSrcFallback = (event, altSrc) => {
+  const handleSrcFallback = (event, altSrc, imageKey) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
-      currentTarget.src = altSrc;
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      if (imageKey === 'imageCap') {
+        currentTarget.src = cardSrcFallbackImg;
+      } else {
+        setShow(s => ({ ...s, logoCap: false }));
+      }
+
+      return;
     }
+
+    currentTarget.src = altSrc;
   };
 
   return (
     <div className={classNames(className, wrapperClassName)} ref={ref}>
       <img
-        className="pgn__card-image-cap"
+        className={classNames('pgn__card-image-cap', { show: show.imageCap })}
         src={src}
-        onError={(event) => handleSrcFallback(event, fallbackSrc)}
+        onError={(event) => handleSrcFallback(event, fallbackSrc, 'imageCap')}
+        onLoad={() => setShow(s => ({ ...s, imageCap: !!src }))}
         alt={srcAlt}
       />
       {!!logoSrc && (
         <img
-          className="pgn__card-logo-cap"
+          className={classNames('pgn__card-logo-cap', { show: show.logoCap })}
           src={logoSrc}
-          onError={(event) => handleSrcFallback(event, fallbackLogoSrc)}
+          onError={(event) => handleSrcFallback(event, fallbackLogoSrc, 'logoCap')}
+          onLoad={() => setShow(s => ({ ...s, logoCap: !!logoSrc }))}
```

Comment:
nit: since we're already in the `!!logoSrc` conditional here, you might just pass `true` here.

Reply:
Good, thanks